app/models/persoon.js

```diff
@@ -7,6 +7,7 @@ export default class PersoonModel extends Model {
   @belongsTo('geboorte', { inverse: null }) geboorte;
   @belongsTo('identificator', { inverse: null }) identificator;
   @belongsTo('geslacht-code', { inverse: null }) geslacht;
+  @belongsTo('nationality') nationality;
```

Comment:
Explicit inverses are recommended since they will be a requirement in the future.
```suggestion
  @belongsTo('nationality', { inverse: null }) nationality;
```

Reply:
Oh, right, this also needs to be a hasMany relationships since a person can have multiple nationalities.

Reply:
Okay, I'm doing changes for the backend. There will be some modifications here too with the relationship changes.